torch/distributed/_tensor/ops/_pointwise_ops.py

```diff
@@ -570,6 +570,7 @@ def linear_pointwise_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> Strategy
     aten._foreach_sqrt.default,
     aten._foreach_sqrt_.default,
     aten._foreach_zero_.default,
+    aten._amp_foreach_non_finite_check_and_unscale_.default,
```

Comment:
`_amp_foreach_non_finite_check_and_unscale_(grads, found_inf, constant)` is very much like `_foreach_mul_` - multiplying grad by constant inplace. The only difference is it will update found_inf in place